fdbserver/BulkLoadUtil.actor.cpp

```diff
@@ -91,33 +93,31 @@ ACTOR Future<std::string> readBulkFileBytes(std::string path, int64_t maxLength)
 				}
 			}
 		}
-
-		return content;
+		return Void();
 	} catch (Error& e) {
 		TraceEvent(SevWarn, "ReadBulkFileBytesError").error(e).detail("Path", path).detail("MaxLength", maxLength);
 		throw;
 	}
 }
 
-ACTOR Future<Void> writeBulkFileBytes(std::string path, StringRef content) {
+ACTOR UNCANCELLABLE Future<Void> writeBulkFileBytes(std::string path, std::shared_ptr<std::string> content) {
 	try {
 		state Reference<IAsyncFile> file = wait(IAsyncFileSystem::filesystem()->open(
 		    abspath(path),
 		    IAsyncFile::OPEN_ATOMIC_WRITE_AND_CREATE | IAsyncFile::OPEN_READWRITE | IAsyncFile::OPEN_CREATE,
 		    0644));
-
 		// For small files (< 1MB), do a single write
-		if (content.size() < 1024 * 1024) {
-			wait(file->write(content.begin(), content.size(), 0));
+		if (content->size() < 1024 * 1024) {
+			wait(file->write(content->data(), content->size(), 0));
```

Comment:
See https://github.com/apple/foundationdb/pull/8526

`wait(uncancellable(file->write(content->data(), content->size(), 0)))`;

Reply:
Yes. I added it to https://github.com/apple/foundationdb/pull/12009

I will redo this PR after the above PR is merged.